rules/msk_module_backend_test.go

```diff
@@ -0,0 +1,154 @@
+package rules
+
+import (
+	"path/filepath"
+	"testing"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/stretchr/testify/require"
+	"github.com/terraform-linters/tflint-plugin-sdk/helper"
+)
+
+func Test_MskModuleBackend(t *testing.T) {
+	rule := NewMskModuleBackendRule()
+
+	tests := []struct {
+		Name     string
+		Files    map[string]string
+		WorkDir  string
+		Expected helper.Issues
+	}{
+		{
+			Name:    "backend doesn't have the team's suffix",
+			WorkDir: filepath.Join("dev-aws", "msk", "pubsub"),
+			Files: map[string]string{"backend.tf": `
```

Comment:
You could use `github.com/MakeNowJust/heredoc` if you wanted to have slightly tidier indentation (e.g. https://github.com/utilitywarehouse/release-tools/blob/2d644fc7749798abeb4f5862c21dbe463f9987bb/internal/cmd/k8s-pr/k8s_pr_test.go#L342)

Reply:
I started with that, but it strips all indentation, and then the columns in the expected ranges do not match

Reply:
> I started with that, but it strips all indentation, and then the columns in the expected ranges do not match

ah right, I guess it does making counting columns a bit more difficult: you have to count from the end of the leading indentation, rather than just count columns directly from the line start